src/main/java/frc/team2412/robot/util/ShooterDataDistancePoint.kt

```diff
@@ -0,0 +1,3 @@
+package frc.team2412.robot.util
+
+data class ShooterDataDistancePoint(val distance: Double, val angle: Double, val power: Double)
```

Comment:
:( More code that VSCode can't parse so intellisense doesn't work for most programmers.  Can you undo this?